saleor/product/models/base.py

```diff
@@ -240,6 +240,15 @@ def get_cost_price(self):
             return stock.cost_price
 
 
+@python_2_unicode_compatible
+class StockLocation(models.Model):
+    location = models.CharField(
```

Comment:
I'd recommend to refactor it to `name`. It will give us better API:
`stock.location.name` instead of `stock.location.location`